src/model/PropertyVerifier.h

```diff
@@ -57,13 +57,13 @@ void checkProperty(OwnerType* object, Getter getter, Signal signal, const char*
 template<typename OwnerType, typename PropertyType>
 static void checkProperty(const PropertyInterface<OwnerType, PropertyType>& property, const char* propertyName)
 {
-    checkProperty(property.object(), property.getter(), property.signal(), propertyName);
+    facelift::checkProperty(property.object(), property.getter(), property.signal(), propertyName);
```

Comment:
we should reduce explicit scoping, not add it. Please remove facelift:: as this is not needed. It only creates mess in the code. Same below. If it causes troubles, use "using" directive.

Reply:
not providing any namespace caused an ambiguous reference error.